package.json

```diff
@@ -24,8 +24,11 @@
   "scripts": {
     "server": "webpack-dev-server",
     "prepare": "WEBPACK_TARGET=lib webpack",
+    "cypress:test": "cypress run --config video=false,trashAssetsBeforeRuns=false",
+    "cypress:open": "cypress open",
     "test": "mocha --compilers js:babel-core/register --require ignore-styles --require ./test/test_helper.js 'test/**/*.@(js|jsx)'",
-    "test:watch": "npm run test -- --watch"
+    "test:watch": "npm run cypress:open & npm run test -- --watch",
+    "test:full": "npm run test && npm run cypress:test"
```

Comment:
I wanted to get all the tests running through Cypress, but I couldn't make it happen...so I added a handful of new testing commands instead. I left `npm run test` alone since the Cypress tests take a while to start up, but they're fine while `watch`ing so I added them there.

Reply:
I'd definitely keep unit tests separate from Cypress. There's no reason to use Cypress machinery for that. It's more likely that developer is already familiar with Mocha, no startup time, it's faster, and Mocha won't die, while Cypress might (I hope not). I'd say it's still very easy to run Cypress, but it's pretty opinionated tool.

Reply:
Agreed that I wouldn't want to depend too much on Cypress! The only reason I had the thought is because Cypress just bundles most of the tools we use by default (Mocha, Chai, and Sinon), so it should be easy to move back and forth. Still, I see what you mean - I'll leave them separate for now.

https://docs.cypress.io/guides/references/bundled-tools.html#